src/screens/SignUpScreen/index.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import RNPickerSelect from 'react-native-picker-select';
+import { StackActions, NavigationActions } from 'react-navigation';
+import { connect } from 'react-redux';
+import { bool, string, func, object } from 'prop-types';
+import { signUp } from '../../actions';
+import { Input, Spinner, BlackButton, WhiteButton, SeparatorView } from '../../components/common';
+import styles from './styles';
+
+class SignUpScreen extends Component {
+  inputRefs = {};
+
+  state = {
+    gender: undefined,
+    items: [
+      {
+        label: 'FEMALE',
+        value: 'female'
+      },
+      {
+        label: 'MALE',
+        value: 'male'
+      },
+      {
+        label: 'OTHER',
+        value: 'other'
+      }
+    ],
+    password: '',
+    email: '',
+    username: ''
+  };
+
+  signUpRequest = async () => {
+    const { username, email, gender, password } = this.state;
+    const { signUp, navigation } = this.props;
+    try {
+      await signUp({
+        username,
+        email,
+        gender,
+        password,
+        passwordConfirmation: password
+      });
+      navigation.navigate('Main');
+    } catch (err) {}
+  };
+
+  goToSignIn = () => {
+    navigation.dispatch(
+      StackActions.reset({
+        index: 0,
+        actions: [NavigationActions.navigate({ routeName: 'SignIn' })]
+      })
+    );
+  };
+
+  renderSpinner = () => {
+    if (this.props.loading) return <Spinner />;
+  };
+
+  render() {
+    const { username, email, gender, password, items } = this.state;
+    return (
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+        <View style={styles.blueCircle} />
+        <View style={styles.yellowCircle} />
+        <Text style={styles.titleText}>TARGET MVD</Text>
+        <View style={{ ...styles.formContainer, marginTop: 150 }}>
```

Comment:
You can set multiple styles like this:
```suggestion
        <View style={[styles.formContainer, { marginTop: 150 }]}>
```

The last items will override the properties that are repeated in previous styles.